examples/nlp/glue_benchmark/glue_benchmark_with_bert.py

```diff
@@ -164,6 +173,8 @@
     "--no_data_cache", action="store_true", help="When specified do not load and store cache preprocessed data.",
 )
 parser.add_argument("--no_shuffle_data", action="store_false", dest="shuffle_data")
+parser.add_argument("--exp_name", default="default", type=str)
+parser.add_argument("--project", default="wand_project", type=str)
```

Comment:
could you leave this by default none, and 
also only add the wand callback if its specified? 
users may not want to use it